src/makeRobotAccountant.js

```diff
@@ -7,6 +7,21 @@
 
 function makeRobotAccountant() {
   // write code here
+  let count = 0;
+
+  return function(firstNumber) {
+    count++;
+
+    const isBroken = count > 3 && count % 2 === 0;
+
+    return function(secondNumber) {
+      if (isBroken) {
+        return `Bzzz... Error!`;
+      }
+
+      return firstNumber + secondNumber;
+    };
```

Comment:
- arrow function would work perfect here, feel free to use it
- don't hesitate to return in ternary if you have just two simple options. It looks better in that case
```suggestion
    return (secondNumber) => (
      isBroken
        ? `Bzzz... Error!`
        :  firstNumber + secondNumber
     );
```